browser/components/MarkdownPreview.js

```diff
@@ -258,7 +258,9 @@ export default class MarkdownPreview extends React.Component {
     theme = consts.THEMES.some((_theme) => _theme === theme) && theme !== 'default'
       ? theme
       : 'elegant'
-    this.getWindow().document.getElementById('codeTheme').href = `${appPath}/node_modules/codemirror/theme/${theme.split(' ')[0]}.css`
+    this.getWindow().document.getElementById('codeTheme').href = theme.startsWith('solarized')
+      ? `${appPath}/node_modules/codemirror/theme/${theme.split(' ')[0]}.css`
```

Comment:
How about using `theme.substr(0, 9);` instead of splitting? Looks a bit clearer to me.

Reply:
Thanks for suggestion.

Personally, I cannot see significant differences between your suggestion and original code from readable perspective. However now this code is only called for solarized dark/light theme, so I think
`${appPath}/node_modules/codemirror/theme/solarized.css`
is the most clear code. What do you think?

Reply:
@yosmoc Oh.. yes. Completely overlooked that.